explorer/files/ncpi-catalog/ncpi-update-utils.ts

```diff
@@ -0,0 +1,42 @@
+import { stringify } from "csv-stringify/sync";
+import fs from "fs";
+import { SOURCE_FIELD_KEY } from "./constants";
+import { dbGapId, NCPIPlatformStudy } from "./entities";
+
+export const sourcePath = "ncpi-catalog/files/dashboard-source-ncpi.tsv";
+
+export function replaceTsv(path: string, data: string[][]): void {
```

Comment:
@jpaten would you mind relocating the method `replaceTsv` to `explorer/app/utils/tsvParser.ts` - it may have other uses in the future...